html/partials/packages.html.haml

```diff
@@ -23,7 +23,7 @@
                   %th.medium Vendor
                   %th Checksum
               %tbody
-                - packages.each do |package|
+                - packages.packages.each do |package|
```

Comment:
I agree that it looks a bit weird. We have the same problem with `services.services` and in a less dramatic form with `config_files.files` etc.

Unfortunately I don't know of a good way to solve that right now...